### DISCUSSION TITLE
Schemacrawler - JPA code generation

### DISCUSSION THREAD

**Discussion by ghost** [schemacrawler/SchemaCrawler#987] [General]
Hello thanks for this wonderful piece of software. 
I am checking if currently, schemacrawler supports generation of JPA entities.

**Comment by sualeh** [comment]
@homihq - not sure if this is a question or a statement.

**Reply by ghost** [reply]
Question - does schemacrawler generate JPA code?

**Reply by sualeh** [reply]
No.
